### DISCUSSION TITLE
Give - Funds and Designations Plugin

### DISCUSSION THREAD

**Discussion by ashokngi** [impress-org/givewp#5859] [Developer Q&A] [answered]
Hi Folks,
I have installed **Give - Funds and Designations Plugin**
Please check attachment on what DB table the **funds value saved** ?
In other words Where I can get the **meta value** of funds in the DB ?
Thanks and regards
![Funds](https://user-images.githubusercontent.com/86282728/123649195-000a1000-d843-11eb-9c0f-cf9fa9dd8b13.png)

**Comment by JasonTheAdams** [comment, marked as answer]
Hi @ashokngi!

I'm not completely sure what you're asking, but donation amounts are tied to specific funds in the `give_revenue` table.

**Reply by ashokngi** [reply]
Thank you Jason, Yes this resolved my issue
